packages/experimental/Link/src/__tests__/Link.test.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { Link } from '..';
+import * as renderer from 'react-test-renderer';
+
+it('Link all props', () => {
+  const tree = renderer.create(<Link url="https://www.bing.com">Link to Bing</Link>).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+it('Link with child', () => {
+  const tree = renderer.create(<Link url="https://www.bing.com">Link to bing</Link>).toJSON();
```

Comment:
These tests don't differ in content, was the intent to put something different here?

Reply:
yes, adding another example her